examples/elementary/simple.py

```diff
@@ -21,8 +25,69 @@ class T:
             P.p0 >> T.t,
             T.t >> P.p1,
         }
+
+        example_markings = {
+            "Start here": {
+                P.p0: {Token()},
+            }
+        }
+
+
+class SimplePetriNetInhibitor(PetriNet):
+    """
+    Elementary Petri net with inhibitor arcs.
+    """
+
+    class Structure:
+        class P:
+            p0 = Place()
+            p1 = Place()
+
+        class T:
+            t = Transition()
+
+        arcs = {
+            (P.p0 >> T.t)(annotation="⃝", guard=inhibitor),
+            (P.p1 >> T.t)(annotation="⃝", guard=inhibitor),
+            T.t >> P.p1,
+        }
+
         example_markings = {
             "Start here": {
                 P.p0: {Token()},
             }
         }
+
+
+class YesNoInhibitor(PetriNet):
+    """
+    You can only choose Yes or No once, and the choice depends on input.
+    """
+
+    class Structure:
+        class P:
+            input = Place()
+            yes = Place()
+            no = Place()
+
+        class T:
+            yes = Transition()
+            no = Transition()
+
+        arcs = {
+            P.input >> T.yes,
+            (P.input >> T.no)(guard=inhibitor),
```

Comment:
Could we also have a factory called `arc(src, dst, **kwargs)` that would return the correct arc type?

I think this DSL demands ppl understand python too well to be productive.